test/indexer/test_hca_indexer.py

```diff
@@ -90,7 +98,7 @@ class BundleAndSize(NamedTuple):
                 try:
                     self._index_bundle(self.new_bundle, manifest, metadata)
 
-                    hits = self._get_all_hits()
+                    hits = self._get_all_hits(translate=False)
```

Comment:
Eliminate the `translate` parameter or explain why it is needed. What's special about this test that requires that no translation is performed?

Reply:
Removed `translate` param, updated test cases accordingly for testing the data as it is in Elasticsearch in its translated (None -> '__null__') state.